frontend/src/routes.tsx

```diff
@@ -7,7 +7,7 @@ import { RouteObject } from "react-router-dom";
 
 const routes: RouteObject[] = [
   {
-    path: "repository/:id",
```

Comment:
We absolutely need to keep this. Otherwise you cannot send a link that points to a repository. Also, whenever you refreshes the browser, you get put back to the search page. 

As a user, i would expect to be able to refresh the page without being put back to the search page.

Reply:
It's done